hooks/post_gen_project.py

```diff
@@ -1,9 +1,137 @@
+import os
+import json
+import shutil
 import subprocess
+from python_terraform import *
+
+YES_OPTIONS, NO_OPTIONS = frozenset(["y"]), frozenset(["n", ""])
+PERM_OPTIONS = frozenset(["push", "pull", "admin"])
+
+def multiple_choices(question):
+    selection = []
+    additional = "Would you like to add another (y/n)? "
+
+    while True:
+        choice = raw_input(question).lower()
+        selection.append(choice)
+        if not check_choice(additional):
+            break
+    return selection
+
+def check_choice(question):
+    while True:
+        choice = raw_input(question).lower()
+        if choice in YES_OPTIONS:
+            return True
+        if choice in NO_OPTIONS:
+            return False
+
+def get_collaborators():
+    # Questions that have multiple layers to them
+    collaborators = []
+    collab_q = "Grant an organizational team access to the repo(y/n)? [n]: "
+    collab = check_choice(collab_q)
+    if collab:
+        while True:
+            teams_q = "What team would you like to grant access? "
+            team = raw_input(teams_q).lower()
+
+            if team != "":
+                while True:
```

Comment:
`while True` is not a great pattern... maybe something like this:

```suggestion
                perm = None
                while perm not in PERM_OPTIONS:
                    perms_q = "What permissions would you like to grant them (push,pull,admin)? "
                    perm = raw_input(perms_q).lower()
                collaborators.append({"name": team, "permission": perm})
```

Reply:
fixed